itou/templates/news/news.html

```diff
@@ -0,0 +1,64 @@
+{% extends "layout/base.html" %}
+{% load theme_inclusion %}
+
+{% block title %}Nouveautés {{ block.super }}{% endblock %}
+
+{% block title_prevstep %}
+    {% include "layout/previous_step.html" with back_url=back_url only %}
+{% endblock %}
+
+{% block title_content %}<h1>Nouveautés</h1>{% endblock %}
+
+{% block content %}
+    <section class="s-section">
+        <div class="s-section__container container" id="news">
+            <ul class="list-group list-group-collapse list-group-flush" id="news-list">
+                {% for month_news in news_page %}
+                    <li class="list-group-item list-group-item-action">
+                        <button class="d-block w-100"
+                                data-bs-toggle="collapse"
+                                data-bs-target="#news-{{ month_news.id }}"
+                                type="button"
+                                aria-expanded="{% if forloop.counter > 1 %}false{% else %}true{% endif %}"
+                                aria-controls="news-{{ month_news.id }}">
+                            <div class="h3 mb-0">
+                                {{ month_news.start_date|date:"F Y"|capfirst }}
+                                <span class="badge badge-sm bg-emploi-light text-info rounded-pill ms-2" aria-label="nombre d'articles">{{ month_news.count_items }}</span>
+                            </div>
+                        </button>
+                        <div class="mt-3{% if forloop.counter > 1 %} collapse{% else %} collapse show{% endif %}" id="news-{{ month_news.id }}" aria-controls="news-{{ month_news.id }}">
+                            {% for news_item in month_news.items.all %}
+                                <div class="row mb-3">
+                                    <div class="col-12 col-md-4{% if news_item.image %} mb-3{% else %} d-none d-md-inline{% endif %}">
+                                        {% if news_item.image %}
+                                            <img src="{{ news_item.image.url }}"
+                                                 loading="lazy"
+                                                 alt="{{ news_item.image_alt_text|default:'' }}"
```

Comment:
image_alt_text sera toujours une `str`.
```suggestion
                                                 alt="{{ news_item.image_alt_text }}"
```